<ui version="4.0" >
 <class>BeeldCompressieUIClass</class>
 <widget class="QMainWindow" name="BeeldCompressieUIClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BeeldCompressieUI</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QGraphicsView" name="screen" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>35</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushComprimeren" >
    <property name="geometry" >
     <rect>
      <x>810</x>
      <y>500</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text" >
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="screenCom" >
    <property name="geometry" >
     <rect>
      <x>490</x>
      <y>34</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600; text-decoration: underline; color:#d2691e;">ORIGINAL&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>670</x>
      <y>10</y>
      <width>112</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:12pt; font-weight:600; text-decoration: underline; color:#d2691e;">COMPRESSED&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>490</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title" >
     <string>Quantization and DCT Properties</string>
    </property>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Start: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>42</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Raise: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinQStart" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinQRaise" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>62</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Blocksize: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBlocksize" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>490</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title" >
     <string>Run-length Encoding Properties</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3" >
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>490</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title" >
     <string>Image Properties</string>
    </property>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>21</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Height: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>43</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Width:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editHeight" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>18</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="editWidth" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4" >
    <property name="geometry" >
     <rect>
      <x>620</x>
      <y>490</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title" >
     <string>Compressed Image Properties</string>
    </property>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>42</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Bytes: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editBytes" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>39</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength" >
      <number>999999999</number>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">kB: &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editKBytes" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>17</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength" >
      <number>999999999</number>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionOpenAs" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionCompress" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHow_do_I" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionOpen" >
   <property name="text" >
    <string>Open (*.raw)</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save Compressed Image (*.comp)</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCompress" >
   <property name="text" >
    <string>Compress</string>
   </property>
   <property name="shortcut" >
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDecompress" >
   <property name="text" >
    <string>Decompress</string>
   </property>
   <property name="shortcut" >
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Alt+F4</string>
   </property>
  </action>
  <action name="actionHow_do_I" >
   <property name="text" >
    <string>How Do I</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenAs" >
   <property name="text" >
    <string>Open Compressed Image (*.comp)</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save Raw Image (*.raw)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
